sys/freebsd/dev_crypto.txt

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/time.h>
+include <opencrypto/cryptodev.h>
+
+resource fd_crypto[fd]
+
+openat$crypto(fd const[AT_FDCWD], file ptr[in, string["/dev/crypto"]], flags flags[open_flags], mode const[0]) fd_crypto
+
+ioctl$CIOCFINDDEV(fd fd_crypto, cmd const[CIOCFINDDEV], arg ptr[in, crypt_find_op])
+ioctl$CIOCGSESSION(fd fd_crypto, cmd const[CIOCGSESSION], arg ptr[in, session_op])
+ioctl$CIOCGSESSION2(fd fd_crypto, cmd const[CIOCGSESSION2], arg ptr[in, session2_op])
+ioctl$CIOCCRYPT(fd fd_crypto, cmd const[CIOCCRYPT], arg ptr[in, crypt_op])
+ioctl$CIOCCRYPTAEAD(fd fd_crypto, cmd const[CIOCCRYPTAEAD], arg ptr[in, crypt_aead])
+ioctl$CIOCFSESSION(fd fd_crypto, cmd const[CIOCFSESSION], arg ptr[in, array[int32]])
+
+crypt_find_op {
+	crid	int32
+	name	array[int8, 32]
+}
+
+session_op {
+	cipher		int32
+	mac		int32
+	keylen		int32
```

Comment:
I believe `keylen` should have type `bytesize[key, int32]`. I'm not 100% sure though that this won't just give the size of the pointer, rather than the size of the array...

Same for mackeylen and other structures below.